src/amitaRJun20/RemoveWordFromArrayList.java

```diff
@@ -0,0 +1,31 @@
+/*Assignment - 34 :  25th July 2020
+Program 1 : 
+Remove all Techno from given ArrayList.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+Hint : contains & remove*/
+
+package amitaRJun20;
+
+import java.util.ArrayList;
+
+public class RemoveWordFromArrayList {
+	
+	void removeElement(ArrayList<String> listOfElements){
+		System.out.println("The ArrayList Input is : "+listOfElements);
+		while(listOfElements.contains("Techno")){
+			listOfElements.remove("Techno");
+		}
+		System.out.println("\nUpdated ArrayList after removing Techno is : "+ listOfElements);
+		}
+
+	public static void main(String[] args) {
+		ArrayList<String> listOfElements = new ArrayList<String>();
```

Comment:
We can use below to create and add elements in Arraylist:
rrayList<String> listOfElements = new ArrayList<String>(Arrays.asList("Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"));

Reply:
Thank you for the suggestion ...i will incorporate this in future code...